src/mainwindow.cpp

```diff
@@ -172,8 +172,8 @@ void MainWindow::setDropShortcut(QKeySequence dropShortCut)
     }
 }
 
-void MainWindow::setup_Action(const char *name, QAction *action, const char *defaultShortcut, const QObject *receiver,
-                              const char *slot, QMenu *menu, const QVariant &data)
+QAction *MainWindow::setup_Action(const char *name, QAction *action, const char *defaultShortcut, const QObject *receiver,
```

Comment:
Is there a reason to return the action here? Looks like no one uses the return value.

Reply:
I used it in an earlier iteration. Forgot to remove it.